logredactor/redacting_filter.py

```diff
@@ -4,34 +4,35 @@
 
 class RedactingFilter(logging.Filter):
     # Do not try and redact the built in values. With the wrong regex it can break the logging
-    ignore_keys = [
+    ignore_keys = {
         'name', 'levelname', 'levelno', 'pathname', 'filename', 'module',
         'exc_info', 'exc_text', 'stack_info', 'lineno', 'funcName', 'created',
         'msecs', 'relativeCreated', 'thread', 'threadName', 'process',
         'processName', 'args',
-    ]
+    }
 
-    def __init__(self, patterns, default_mask='****'):
+    def __init__(self, patterns, default_mask='****', mask_keys={}):
```

Comment:
[Python gotacha](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments) using mutable as defaults

Reply:
Done, thanks for the comment:)